README.md

```diff
@@ -176,7 +176,7 @@ Run `npm run build` to compile the code to Javascript.
 
 Run `node dist/examples/runFasterCode.js` (to use GraphQL) or `node dist/examples/runOldCode.ts` (to use REST calls), to run the example program. It requires internet access, since it calls the GitHub API. It will take a couple minutes to complete. Some of the output includes the word "ERROR", so don't panic.
 
-Ensure to lint your code by running `npm run lint` before submitting any code for review. `npm run lint:fix` will automatically fix any errors.
+Husky now runs linting checks pre-commit. Either manually fix the errors or run `npm run lint:fix` to automatically fix any errors.
```

Comment:
I would favor still mentioning how to manually run the linting, in case folks want to do it before attempting a commit.

Reply:
@redgardner : Also, it looks like there is a bug in husky where sometimes the hooks won't run if you are using an IDE (like vscode in my case) or visual system like SourceTree. We should probably mention that here too, so people realize that the hook not a sure thing. (Which is super frustrating, since that's kind of the point.)

Reply:
Reference: https://github.com/typicode/husky/issues/639